### ISSUE TITLE
Hook up FragmentShader API in painting.dart to CanvasKit

### ISSUE THREAD

**Issue by clocksmith** [flutter/flutter#85238]
blocked by https://github.com/flutter/flutter/issues/85237

**Comment by skreborn** [comment]
@clocksmith Since the blocking dependency has been closed for a while, is there anything particular stopping this from going forward, or is it "just" that it needs an actual implementation?

**Comment by jonahwilliams** [comment]
Closing in favor of https://github.com/flutter/flutter/issues/114260. This is "done" but requires updates to the engine canvaskit build that have no propogated yet.
